xlr/converters/src/__tests__/ts-to-common.test.ts

```diff
@@ -656,4 +656,28 @@ describe('Variable Exports', () => {
 
     expect(XLR).toMatchSnapshot();
   });
+
+  it('Aliased Arrow function exports its own name', () => {
+    const sc = `
+      interface Bar {
+        foo: string
+        fuz: number
+      }
+
+      export const foo = (input: number): Bar => {
```

Comment:
You _technically_ don't need to export this. Since `convertSourceFile` will convert every exported node by default  you just need to export `baz` for the to the test to check what it needs to check.

Reply:
sure, but exporting both is more representative of #21 since that was based on exporting `size` and `length`